core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2813,13 +2813,19 @@ public String toString() {
               OperatingSystemMXBean operatingSystemMXBean =
                   (OperatingSystemMXBean) ManagementFactory.getOperatingSystemMXBean();
               return operatingSystemMXBean.getTotalPhysicalMemorySize() * 2 / 3;
-            } catch (Exception e) {
+            } catch (Throwable e) {
               // The package com.sun.management may not be available on every platform.
               // fallback to a reasonable size.
               return "1GB";
             }
           }, "2/3 of total system memory, or 1GB if system memory size cannot be determined")
-          .setDescription("Memory capacity of each worker node.")
+          .setDescription("Memory capacity of each worker node. "
+                  + "It is recommended to set this value explicitly. "
+                  + "If you are running Alluxio workers in a containerized environment, "
+                  + "your Java runtime may return incorrect system memory "
+                  + "(i.e. the host memory instead of the container memory). "
+                  + "You should upgrade your JDK version according to "
```

Comment:
Why higher version JDK doesn't have this issue? I don't think the link recommend the upgrade, or I just didn't find the right place?